lib/src/pdf_document_proxy.dart

```diff
@@ -173,11 +173,19 @@ class PDFDocumentProxy {
   Future<List<OutlineItem>> getOutline() {
     JsObject promise = _jsInternal.callMethod('getOutline', []) as JsObject;
 
-    return _promiseToFuture<List<OutlineItem>>(promise,
-        transform: (value) {
-      if (value is List<JsObject>) {
```

Comment:
This conditional was added in in [this recent PR](https://github.com/Workiva/pdfjs_dart/pull/35/files?file-filters%5B%5D=.dart#diff-5eea3942701854603dfd3fac258a731278fae1f63946acd4f7f869ef78a6c2a7R178), but turns out that logic was not sufficient, as you can't cast directly from a JsArray<dynamic> to a List<JsObject>. So more parsing/typesafety is needed